### DISCUSSION TITLE
Login SignalK

### DISCUSSION THREAD

**Discussion by Kaythaltes** [bareboat-necessities/lysmarine_gen#124] [Q&A] [answered]
To make changes in SignalK it's asking to login. But I user/changme is not working. How to get in?

**Comment by mgrouch** [comment, marked as answer]
First time you login into SignalK it will ask you to set up user and password.

**Reply by visailor** [reply]
So I think I've done that, but now I've forgotten my password... What can I do to reset it or bypass it?

**Reply by mgrouch** [reply]
You need to delete security.json in /home/signalk/.SignalK and restart SignalK
